engines/web/web.gemspec

```diff
@@ -0,0 +1,17 @@
+$LOAD_PATH.push File.expand_path('lib', __dir__)
+
+require "web/version"
+
+Gem::Specification.new do |s|
+  s.name        = "web"
+  s.version     = Web::VERSION
+  s.authors     = ["developers@ofn"]
+  s.summary     = "Web domain of the OFN solution."
+
+  s.files = Dir["{app,config,db,lib}/**/*"] + ["LICENSE.txt", "Rakefile", "README.rdoc"]
+  s.test_files = Dir["test/**/*"]
+
+  s.add_dependency "rails", "~> 3.2.22.5"
+
+  s.add_development_dependency "sqlite3"
```

Comment:
IMO this should be `pg` instead.

Reply:
I have removed this entry and the rails one as well. This was auto generated. I dont think we need this.